build.gradle

```diff
@@ -52,15 +53,10 @@ android {
         }
     }
 
-    /*
-    // call regular ndk-build(.cmd)ol script from app directory
-    task ndkBuild(type: Exec) {
-        commandLine 'ndk-build', '-C', file('src/main').absolutePath
+    externalNativeBuild {
```

Comment:
adding this part gives me this error (rebased to master - got other errors when using directly):

Reply:
```
2 warnings generated.
[x86_64] Compile        : gnuGo-3.8 <= endgame.c
[x86_64] Compile        : gnuGo-3.8 <= patterns.c
[x86_64] Compile        : gnuGo-3.8 <= transform.c
[x86_64] Compile        : gnuGo-3.8 <= sgf_utils.c
[x86_64] Compile        : gnuGo-3.8 <= sgfnode.c
[x86_64] Compile        : gnuGo-3.8 <= sgftree.c
[x86_64] Compile        : gnuGo-3.8 <= gg_utils.c
[x86_64] Compile        : gnuGo-3.8 <= random.c
[x86_64] Compile        : gnuGo-3.8 <= winsocket.c
/home/ligi/git/gobandroid-ai-gnugo/src/main/jni/project/gnugo-3.8/utils/winsocket.c:238:12: warning: implicit declaration of function 'winsocket_vfprintf' is invalid in C99 [-Wimplicit-function-declaration]
  result = winsocket_vfprintf(file, format_string, arguments);
           ^
1 warning generated.
[x86_64] SharedLibrary  : libgnuGo-3.8.so

> Task :mergeReleaseNativeLibs FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':mergeReleaseNativeLibs'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.Workers$ActionFacade
   > More than one file was found with OS independent path 'lib/arm64-v8a/libgnuGo-3.8.so'. If you are using jniLibs and CMake IMPORTED targets, see https://developer.android.com/studio/preview/features#automatic_packaging_of_prebuilt_dependencies_used_by_cmake

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

BUILD FAILED in 4m 46s
28 actionable tasks: 17 executed, 11 up-to-date
```